src/app/shared/addressHelper.ts

```diff
@@ -71,5 +68,12 @@ export const generateHTMLofAddress = (address: PostalAddressClass): string => {
   if (address.addressRegion) html += `${address.addressRegion}<br>`;
   html += `${address.postalCode}`;
 
+  const countryName = address.addressCountry
+    ? countryMap[address.addressCountry]
+    : undefined;
+  if (countryName && address.addressCountry !== "GB") {
```

Comment:
could we check if country code is non GB _before_ doing the lookup? slight optimisation so we only do the lookup for intl' addresses